templates/zerver/help/view-someones-profile.md

```diff
@@ -21,6 +21,15 @@ streams for which [you can see all subscribers](/help/stream-permissions).
     You can also click on a user's profile picture or name and then
     click **View full profile**.
 
+{tab|mobile}
+
+1. Tap on a user's profile picture or name.
+
+!!! Tip ""
+
+    If you are viewing a private message, you can also tap the **ⓘ** icon
```

Comment:
Is there a more robust way to put in the same icon as we have in the mobile app? It's not a huge deal, but this doesn't look quite right.

Reply:
It seems like the exact same icon is not available in this codebase, but maybe we could pull it from zulip-mobile?

We could use a similar one that's available though:
<img width="531" alt="image" src="https://user-images.githubusercontent.com/2343554/178609431-312d2784-e018-447d-b36d-8e6a0c7545a1.png">

Reply:
Let's try to figure our how to use the correct icon, not one that's somewhat similar. If you aren't sure how best to do it from a technical perspective, could you start a thread on CZO?

Reply:
Sounds good!
https://chat.zulip.org/#narrow/stream/19-documentation/topic/need.20mobile.20icons.20for.20docs